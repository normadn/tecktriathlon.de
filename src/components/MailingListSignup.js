import React from "react";
import SignUpForm from "./SignUpForm";
import ContentWrapper from "./ContentWrapper";

import circuits from "../assets/img/backgrounds/circuitsGrey.png";

const MailingListSignup = (props) => (
  <section
    css={{
      width: "100%",
      color: "white",
      minHeight: "130px",
      display: "flex",
      alignItems: "center",
      justifyContent: "center",
      background: `#11203F url(${circuits}) no-repeat center center`,
      backgroundSize: "cover"
    }}
  >
    <ContentWrapper>
      <p
        data-cy="mailing-list-cta"
        css={{
          fontWeight: 700,
          marginTop: "20px",
          marginBottom: "28px",
          fontSize: "22px",
          "@media screen and (max-width: 860px)": {
            fontSize: "16px"
          },
          lineHeight: "28px",
          textAlign: "center"
        }}
      >
        Teck
      </p>
    </ContentWrapper>
  </section>
);

export default MailingListSignup;
