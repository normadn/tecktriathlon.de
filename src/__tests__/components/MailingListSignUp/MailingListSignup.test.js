import React from "react";
import MailingListSignup from "../../../components/MailingListSignup";
import { shallow } from "enzyme";

describe("MailingListSignup", () => {
  it("renders correctly", () => {
    expect(shallow(<MailingListSignup />)).toMatchSnapshot();
  });
});
