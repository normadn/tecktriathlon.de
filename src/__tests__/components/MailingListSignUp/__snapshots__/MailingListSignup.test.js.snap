// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MailingListSignup renders correctly 1`] = `
<section
  css={
    Object {
      "alignItems": "center",
      "background": "#C12232 url(test-file-stub) no-repeat center center",
      "backgroundSize": "cover",
      "color": "white",
      "display": "flex",
      "justifyContent": "center",
      "minHeight": "270px",
      "width": "100%",
    }
  }
>
  <_default>
    <p
      css={
        Object {
          "@media screen and (max-width: 860px)": Object {
            "fontSize": "16px",
          },
          "fontSize": "22px",
          "fontWeight": 700,
          "lineHeight": "28px",
          "marginBottom": "28px",
          "marginTop": "20px",
          "textAlign": "center",
        }
      }
      data-cy="mailing-list-cta"
    >
      Sign up for our mailing list to receive important QHacks announcements!
    </p>
    <Apollo(SignUpForm)
      center={true}
      dataCyButton="mailing-list-button"
      dataCyInput="mailing-list-input"
      hoverBackgroundColor="#00205b"
    />
  </_default>
</section>
`;
